### PR TITLE
Use new button id to click add new work button

### PR DESCRIPTION
**Pull request by hackartisan**
Fixes the build

### CODE REVIEW

**Review by elrayle — APPROVED** [review]

**Review by jcoyne — CHANGES_REQUESTED** [review]
Shouldn't we test that this link is accessible to non-sighted users?

**Comment by hackartisan** [comment]
@jcoyne the test was failing because the capitalization was changed in another PR while the one that added this test was in review.

**Comment by jcoyne** [comment]
@hackartisan Yes. My suggestion be to change that capitalization rather than ignore it.

File: spec/features/create_work_spec.rb [hunk 1]
```diff
@@ -139,7 +139,7 @@
     before do
       sign_in user
       click_link 'Works'
-      click_link "Add new work"
+      find('#add-new-work-button').click
```
[the remark this answers is not in the document]
Comment (jcoyne) [reply spec/features/create_work_spec.rb:142]: This allows it to match https://github.com/samvera/hyrax/blob/bba8958bc306a63ff97005658a66f0bdfacbf348/config/locales/hyrax.en.yml#L60

**Comment by hackartisan** [comment]
@jcoyne I accepted your suggestion

**Review by jcoyne — APPROVED** [review]

**Pull request merged**